js/util.js

```diff
@@ -402,6 +402,82 @@
     }
 
 
+    /*
+     * Motion mini-module.
+     *
+     * Call startTrackingMotion(). The direction property will be
+     * updated.
+     *
+     * Events:
+     *
+     */
+    var motionModule  = (function () {
+        var direction = ""
+            motionModule;
```

Comment:
There should be a comma after `direction = ""`. I'm gonna be honest and say I have no idea why this works right now as it is. Sorry about my strange JSLint-induced habit of having _one_ big `var` statement at the top of every function. Multiple `var` statements are fine.